user/admin.py

```diff
@@ -1,7 +1,9 @@
+"""Registers the custom User and the CustomUserAdmin.
+"""
```

Comment:
Start a new line after the """.
For good practice of our style, please see: https://github.com/numpy/numpy/blob/master/numpy/polynomial/polynomial.py
and 
https://github.com/ahaselsteiner/viroconcom/blob/master/viroconcom/_n_sphere.py

Reply:
For me this description is not so clear yet: what is a "custom User"? and why is User from .models and CustomUserAdmin from .forms (and not from the same module)?